src/cljs/ataru/virkailija/editor/handlers.cljs

```diff
@@ -108,6 +108,15 @@
           (update-in text-field-path into [component])
           (update-in (drop-last text-field-path) set-non-koodisto-option-values)))))
 
+(reg-event-db
+  :editor/aseta-lisäkysymys-arvon-perusteella-operaattori
+  (fn [db [_ path value]]
+    (let [condition-path (current-form-content-path db [path :options 0 :condition])]
+      (-> db
```

Comment:
Jos on vain yksi operaatio, niin suosisin itse threading -makron välttämistä eli tässä tapauksessa käyttäisin muotoa `(update-in db condition-path update-fn)`.